tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker;
+
+import alluxio.IntegrationTestUtils;
+import alluxio.LocalAlluxioClusterResource;
+import alluxio.Version;
+import alluxio.WorkerStorageTierAssoc;
+import alluxio.master.MasterContext;
+import alluxio.metrics.MetricsSystem;
+import alluxio.rest.TestCaseFactory;
+import alluxio.util.CommonUtils;
+import alluxio.worker.block.BlockStoreMeta;
+import alluxio.worker.block.BlockWorker;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.net.InetSocketAddress;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.SortedMap;
+
+/**
+ * Test cases for {@link AlluxioWorkerRestServiceHandler}.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({AlluxioWorker.class, BlockWorker.class, BlockStoreMeta.class})
+@Ignore
```

Comment:
Jenkins timeouts when running this test (same for the one you commented below), is this also the reason why you put ignore?